apigentools/commands/init.py

```diff
@@ -16,6 +16,54 @@
 
 
 class InitCommand(Command):
+    def __init__(self, config, args):
+        self.config_file_json = {
```

Comment:
It's a great idea to move these out of the method code 👍 . Let's put them to the class definition level. They're not really instance-specific, they'll always be the same for all instances of this class.